MetamodelLD.md

```diff
@@ -13,8 +13,8 @@ Vanuit een Linked Data perspectief is dit bijzonder. Een kernmerk van Linked Dat
 Zo leidt een MIM objecttype "Schip" tot de volgende weergave in RDF:
 
 <pre class='ex-turtle'>
-@prefix vb: &lt;http://bp4mc2.org/voorbeeld/> .
-@prefix mim: &lt;http://bp4mc2.org/def/mim#> .
+@prefix vb: &lt;http://modellen.mim-standaard.nl/voorbeeld/> .
```

Comment:
@architolk ik dacht dat we hadden besproken deze aanpassing nog niet door te voeren in deze versie ivm breaking change

Reply:
Ja daar werd zo over gesproken. Maar is het echt breaking en heeft het aanzienlijke ongewenste impact? Zo ja dan terug veranderen. Zo nee dan is het weer een stap in de goede richting.

Reply:
Dit is breaking, omdat er een nieuwe namespaces wordt geintroduceerd. Dit is vergelijkbaar met het introduceren van een nieuwe namespaces voor alle elementen in een XML schema.
Iedereen die nu gebruikmaakt van MIM-LD zal dus al hun modellen hierop aan moeten passen.

Reply:
Sinds MIM 1.1.1 is de officiële namespace http://modellen.mim-standaard.nl/def/mim. Dit wat ook al zo verwerkt in mim.ttl en mim-shapes.ttl. Het was alleen nog niet doorgevoerd in enkele voorbeelden en verwijzingen. Breaking zou m.i. dus zijn om dit weer terug te draaien. Het was alleen op enkele voorbeeldplekken nog niet aangepast, dat is nu gefixeerd. Wat we m.i. hadden afgesproken dat we het niet zouden aanpassen in de delen van de standaard die als "conforming" gelden, en dat daar de namespace niet in thuis zou horen.

Reply:
komen jullie er samen uit?

Reply:
We hebben besproken dat we de nieuwe namespace laten staan en een duidelijke vermelding maken in het versielog dat de namespace nu overal is doorgevoerd. In mijn ogen is dit een breaking change. Marco ziet dat anders, gezien volgens hem de nieuwe ontologie al in de vorige versie is geintroduceerd. Alleen deze is toen niet goed verwerkt in de normatieve tekst van de standaard.
Afin, we gaan kijken of dit reacties van gebruikers oplevert.

Reply:
kan die nu gemerged worden? Inclusief de toegevoegde vermelding in het versielog.

Reply:
@architolk wat staat er nu nog open?